BudgetHelper/src/main/java/com/budgethelper/service/impl/CategoryServiceImpl.java

```diff
@@ -3,36 +3,35 @@
 import com.budgethelper.dao.CategoryDao;
 import com.budgethelper.model.Category;
 import com.budgethelper.service.CategoryService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
-
 import java.util.List;
+import org.springframework.stereotype.Service;
 
 @Service
-@Transactional
-public final class CategoryServiceImpl implements CategoryService {
+public class CategoryServiceImpl implements CategoryService {
 
-	@Autowired
-	private CategoryDao categoryDao;
+	private final CategoryDao categoryDao;
 
-	@Override
-	public void save(final Category category) {
-		this.categoryDao.add(category);
+	public CategoryServiceImpl(final CategoryDao categoryDao) {
+		this.categoryDao = categoryDao;
 	}
```

Comment:
Please use the common formating for this file.

Reply:
I don't understand. Can you explain that?

Reply:
Please, open this file in your environment and I'll see that some lines of it have redundant whitespaces.

Reply:
done